reporter/src/funTest/assets/notice-template-reporter-expected-results-summary

```diff
@@ -0,0 +1,82 @@
+This project contains or depends on third-party software components pursuant to the following licenses:
```

Comment:
Just as a remark, it would have been nice to see in this commit the diff between the expected results of the original `NoticeSummaryReporter` and the accordingly templated `NoticeTemplateReporter`, but I acknowledge that's difficult to accomplish. However, did you inspect that diff manually to verify against any issues?

Reply:
Yes, the only difference is "MIT WITH Libtool-exception" which is not handled by the `NoticeSummaryReporter`.